### PR TITLE
Bump RefStack to OpenStack Powered Platform 2022.11

### PR DESCRIPTION
**Pull request by priteau**

### CODE REVIEW

**Review by Alex-Welsh — APPROVED** [review]

**Comment by Alex-Welsh** [comment]
Might be worth waiting until SMS is back to merge so we can check everything passes the new CI tests

**Comment by markgoddard** [comment]
All non-Rocky jobs passed. Should we wait for Rocky to stabilise before merging?

**Review by markgoddard — APPROVED** [review]

**Pull request merged**
